generators/pkg/generator/generate.go

```diff
@@ -41,6 +46,14 @@ func GenerateTask(github *GitHub) *v1beta1.Task {
 		labels = make(map[string]string)
 	}
 	labels["generator.tekton.dev"] = github.Name
+
+	// DOCKER_CONFIG is used for kaniko to detect docker credential
+	env := &github.Spec.Steps[0].Env
+	*env = append(*env, corev1.EnvVar{
+		Name:  "DOCKER_CONFIG",
+		Value: "/tekton/home/.docker/",
+	})
```

Comment:
These look like options that are step specific. I think we should remove these, letting users set them manually in their steps if they want it. What do you think?

Reply:
I agree. The reason why I add it here is that when I apply the resources on the cluster, the pipelinerun failed since the DOCKER_CONFIG is required to allow kaniko to detect docker credential.